index.bs

```diff
@@ -2669,20 +2669,13 @@ attributes.
     ::  If this member is present, eligible authenticators are filtered to only authenticators attached with the
         specified [[#enum-attachment]]. The value SHOULD be a member of {{AuthenticatorAttachment}} but [=client platforms=] MUST ignore unknown values, treating an unknown value as if the [=map/exist|member does not exist=].
 
-    :   <dfn>requireResidentKey</dfn>
-    ::  Note: This member is retained for backwards compatibility with WebAuthn Level 1 but is deprecated in favour of {{residentKey}}.
-        {{requireResidentKey}} is ignored if the caller supplies {{residentKey}} and the latter is understood by the [=client=].
-        Otherwise, {{requireResidentKey}}'s value is used. Note that {{requireResidentKey}}'s value defaults to [FALSE].
-
-        If used in absence of {{residentKey}}, it describes the [=[RP]=]'s requirements regarding [=client-side discoverable credentials=].
-        If {{requireResidentKey}} is set to [TRUE], the authenticator MUST create a [=client-side discoverable public key credential source=]
-        when creating a [=public key credential=].
-
     :   <dfn>residentKey</dfn>
-    ::  Note: This member supersedes {{requireResidentKey}}. If both are present and the [=client=] understands {{residentKey}}, then
-        {{residentKey}} is used and {{requireResidentKey}} is ignored.
+    ::  Specifies the extent to which the [=[RP]=] desires to create a [=client-side discoverable credential=]. For historical reasons the naming retains the deprecated “resident” terminology. The value SHOULD be a member of {{ResidentKeyRequirement}} but [=client platforms=] MUST ignore unknown values, treating an unknown value as if the [=map/exist|member does not exist=]. If no value is given then the effective value is {{ResidentKeyRequirement/required}} if {{requireResidentKey}} is [TRUE] or {{ResidentKeyRequirement/discouraged}} if it is [FALSE] or absent.
 
-        The value SHOULD be a member of {{ResidentKeyRequirement}} but [=client platforms=] MUST ignore unknown values, treating an unknown value as if the [=map/exist|member does not exist=]. See {{ResidentKeyRequirement}} for the description of {{residentKey}}'s values and semantics.
+        See {{ResidentKeyRequirement}} for the description of {{residentKey}}'s values and semantics.
+
+    :   <dfn>requireResidentKey</dfn>
```

Comment:
> [=[RPS]=] SHOULD set it to [TRUE] if, and only if, {{residentKey}} is set to {{ResidentKeyRequirement/required}}.

Didn't understand why this line was required? Can we just say that residentKey supersedes requireResidentKey as was the case earlier?

Reply:
This will allow RPs to continue to work with user-agents that only implement WebAuth level one. Preferred will never result in a resident credential, but required will.

Reply:
Got it. Thanks.